opencga-app/src/main/java/org/opencb/opencga/app/cli/main/io/JsonOutputWriter.java

```diff
@@ -36,26 +28,12 @@
  */
 public class JsonOutputWriter extends AbstractOutputWriter {
 
-    private ObjectMapper objectMapper;
-
     public JsonOutputWriter() {
         super();
-        initObjectMapper();
     }
 
     public JsonOutputWriter(WriterConfiguration writerConfiguration) {
         super(writerConfiguration);
-        initObjectMapper();
-    }
-
-    private void initObjectMapper() {
-        // Same options as in OpenCGAWSServer
-        objectMapper = new ObjectMapper();
-        objectMapper.addMixIn(GenericRecord.class, GenericRecordAvroJsonMixin.class);
-        objectMapper.addMixIn(VariantStats.class, VariantStatsJsonMixin.class);
```

Comment:
Are we loosing VariantStatsJsonMixin and GenotypeJsonMixin in purpose?

These could be added to the jackson utils. Then, check the OpenCGAWSServer, as it is adding them manually 

```java
        jsonObjectMapper = getExternalOpencgaObjectMapper();
        jsonObjectMapper.addMixIn(GenericRecord.class, GenericRecordAvroJsonMixin.class);
        jsonObjectMapper.addMixIn(VariantStats.class, VariantStatsJsonMixin.class);
        jsonObjectMapper.addMixIn(Genotype.class, GenotypeJsonMixin.class);
```

Reply:
That was because the VariantStatsJsonMixin and GenotypeJsonMixin could not be seen by the module 'core'. I have now moved those mixins to 'core' and added them to JacksonUtils.